classes/fakeroot.oeclass

```diff
@@ -3,7 +3,7 @@
 ## By inheriting this class, it is ensures that the native version of fakeroot
 ## build, is used instead of fakeroot from the build system.
 
-CLASS_DEPENDS += "native:fakeroot"
+CLASS_DEPENDS += "native:fakeroot native:libacl1-dev"
```

Comment:
Uh, why is that libacl1-dev and not just libacl1?  We really should normally not depend on lib*-dev in DEPENDS.  All lib* packages is responsible for pulling in their own lib*-dev.  If they don't do that, they are broken.